pkg/meta/tkv.go

```diff
@@ -3756,3 +3780,89 @@ func (m *kvMeta) loadDumpedACLs(ctx Context) error {
 		return nil
 	})
 }
+
+type kvDirHandler struct {
+	dirHandler
+}
+
+func (h *kvDirHandler) Delete(name string) {
+	h.Lock()
+	defer h.Unlock()
+	h.dirHandler.delete(name)
+}
+
+func (m *kvMeta) newDirHandler(inode Ino, plus bool, entries []*Entry) DirHandler {
+	s := &kvDirHandler{
+		dirHandler: dirHandler{
+			inode:       inode,
+			plus:        plus,
+			initEntries: entries,
+			fetcher:     m.getDirFetcher(),
+			batchNum:    DirBatchNum["kv"],
+		},
+	}
+	s.batch, _ = s.fetch(Background, 0)
```

Comment:
pass the `ctx` instead of `Background`? And the error should be returned?

Reply:
error can be ignore, will retry in the following readdir